rero_ils/filter.py

```diff
@@ -134,3 +136,18 @@ def empty_data(data, replacement_string='No data'):
     else:
         msg = '<em class="no-data">{0}</em>'.format(replacement_string)
         return Markup(msg)
+
+
+def lib_url(user):
+    """Return organisation viewcode.
+
+    :param user - invenio user to get viewcode
+    """
+    viewcode = current_app.config.get('RERO_ILS_SEARCH_GLOBAL_VIEW_CODE')
+    patron = Patron.get_patron_by_user(user)
+    if patron is not None:
+        viewcode = patron.get_organisation().get('code')
+    return '{base_url}/{viewcode}'.format(
```

Comment:
If it is possible try to use `url_for`, but I'm not sure that it is available.

Reply:
No, I tried with url_for but it's not available here.